From: Simon McVittie <smcv@debian.org>
Date: Sun, 12 Jun 2016 09:59:34 +0100
Subject: Link libselinux into bwrap (if enabled) with LDADD, not LDFLAGS

As explained in bubblewrap commit f6ca3690, libraries should
always go in LDADD and not LDFLAGS, because the order of arguments
to the linker matters. Many distributions' linkers are tolerant
enough that it doesn't matter in practice, but it matters for
static linking, and it might also matter on Ubuntu.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 7bbfef7..ddc3e37 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -81,7 +81,7 @@ include tests/Makefile.am.inc
 bwrap_PROGRAMS = flatpak-bwrap
 flatpak_bwrap_SOURCES = $(bwrap_SOURCES)
 flatpak_bwrap_CFLAGS = $(bwrap_CFLAGS)
-flatpak_bwrap_LDFLAGS = $(bwrap_LDFLAGS)
+flatpak_bwrap_LDADD = $(bwrap_LDADD)
 bwrapdir = $(libexecdir)
 include bubblewrap/Makefile-bwrap.am.inc
 
